client/src/components/app/app.js

```diff
@@ -1,26 +1,34 @@
-import React from 'react';
-import './app.css';
-
+import React, { useState } from 'react';
 import { BrowserRouter as Router } from 'react-router-dom';
 
+import { UserContext, ServiceContext } from '../context';
 import { navigation } from '../router/navigation';
+import TestService from '../../service/TestService';
+
 import Header from '../header/Header';
+import ErrorBoundry from '../error-boundary/ErrorBoundary';
 
-const isAuth = false;
-const user = {
-  isExpert: false,
-  isAdmin: false,
-};
+import './app.css';
 
 const App = () => {
-  const { routes, links } = navigation(isAuth, user);
+  const [user, setUser] = useState(null);
+
+  const [service, setService] = useState(new TestService());
+  const { Routes, links } = navigation(user);
+
   return (
-    <Router>
-      <div className="wrapper">
-        <Header>{links}</Header>
-        <div>{routes}</div>
-      </div>
-    </Router>
+    <ServiceContext.Provider value={service}>
```

Comment:
same for service context, but it's a get only context

Reply:
resolved